src/libs/PolicyUtils.js

```diff
@@ -92,4 +92,13 @@ function isExpensifyTeam(email) {
     return emailDomain === CONST.EXPENSIFY_PARTNER_NAME || emailDomain === CONST.EMAIL.GUIDES_DOMAIN;
 }
 
-export {hasPolicyMemberError, hasPolicyError, hasPolicyErrorFields, hasCustomUnitsError, getPolicyBrickRoadIndicatorStatus, shouldShowPolicy, isExpensifyTeam};
+/**
+ * Is the user an admin of the policy?
+ * @param {Object} policy
+ * @returns {Boolean}
+ */
+function isPolicyAdmin(policy) {
```

Comment:
Since this method is only called once, and it only executes a single line of logic, I think it doesn't make sense to have it be a method.

If someone in the future wants to make it a method that is referenced multiple times, they can do it then.

Reply:
Oh yeah good point. I'm probably overly fond of util methods 😂